modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -168,6 +173,46 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
           }
         end
 
+        def get_phase_type_indicator_array(data)
+          return if data[:phase_type_change_ind].nil?
+
+          (data[:phase_type_change_ind]).split('')
+        end
+
+        def ever_phase_back(data)
+          return if data[:phase_type_change_ind].nil?
+
+          pt_ind_array = get_phase_type_indicator_array(data)
+          pt_ind_array.first.to_i > pt_ind_array.last.to_i
+        end
+
+        def get_evss_claim(data)
+          evss_id = data[:benefit_claim_id]
+          ClaimsApi::EVSSClaim.new(evss_id: evss_id, data: data)
```

Comment:
Hmmm, I'm not sure we're gonna want to do this because one of the big goals with V2 is to completely disconnect from EVSS. (it gets permanently sunset 1-yr from today).  

If we start adding new logic into old objects explicitly named EVSS, that's going to lead to confusion for devs in the future when EVSS doesn't even exist.

There's already a simple service that exists for trying to map the value of a BGS claim `status` attribute to the string values we want our API to provide. ([REF](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/claims_api/bgs_claim_status_mapper.rb))

Maybe we can encapsulate this new mapping logic you need over in there?